dynamodb.tf

```diff
@@ -27,8 +27,13 @@ module "dynamodb" {
   global_secondary_index_map = var.dynamodb_global_secondary_index_map
   local_secondary_index_map  = var.dynamodb_local_secondary_index_map
 
-  # We can always extend the defaults and add them
-  enable_autoscaler = false
+  autoscale_write_target       = var.dynamodb_autoscale_write_target
+  autoscale_read_target        = var.dynamodb_autoscale_read_target
+  autoscale_min_read_capacity  = var.dynamodb_autoscale_min_read_capacity
+  autoscale_max_read_capacity  = var.dynamodb_autoscale_max_read_capacity
+  autoscale_min_write_capacity = var.dynamodb_autoscale_min_write_capacity
+  autoscale_max_write_capacity = var.dynamodb_autoscale_max_write_capacity
+  enable_autoscaler            = var.dynamodb_enable_autoscaler ? "true" : "false"
```

Comment:
Isn't that also supposed to be a boolean instead of a string?

Reply:
@cytopia https://github.com/cloudposse/terraform-aws-dynamodb/blob/80c2c516265a379a06258d8bf81892602d151637/variables.tf#L137

Reply:
ok